docs/development/database_schema.md

```diff
@@ -208,10 +208,11 @@ But hash collisions are still possible, and by treating event IDs as room
 scoped, we can reduce the possibility of a hash collision. When scoping
 `event_id` in the database schema, it should be also accompanied by `room_id`
 (`PRIMARY KEY (room_id, event_id)`) and lookups should be done through the pair
-`(room_id, event_id)`.
+`(room_id, event_id)`. Another benefit of scoping `event_ids` to the room is
```

Comment:
I'm not sure I follow this reasoning. Is the point that we can do this with a single `DELETE FROM ... WHERE ...` rather than having to use a subselect or similar which joins to the events table?

Reply:
@DMRobertson Seems to be the case so we don't have to do this: https://github.com/matrix-org/synapse/blob/fa2f3d8d0c915c6ebd84ef53526ac04c91fd4a19/synapse/storage/databases/main/purge_events.py#L388-L412

From the chapter sync, it was one of the useful benefits people liked from pairing up `(room_id, event_id)`.